docs/keto/guides/migrating-legacy-policies.mdx

```diff
@@ -0,0 +1,112 @@
+---
+id: migrating-legacy-policies
+title: Migrating policies from 0.5 to 0.6
+---
+
+0.6 release makes Ory Access Control Policy DSL modeled after AWS IAM Policies obsolete. This guide will help you to rewrite your policies in to [relation-tuples](https://www.ory.sh/docs/keto/concepts/relation-tuples). You can read [The Evolution of Ory Keto: A Global Scale Authorization System](https://www.ory.sh/keto-zanzibar-evolution/) blogpost to understand a benefits of 0.6 release
+
+## Legacy rules example
+
+The policy below allows `Alice` and `Bob` to create/read/modify/delete `blog_posts:my-first-blog-post`, `blog_posts:2`, and `blog_posts:3`.
+
+```json
+{
+  "subjects": ["alice", "bob"],
+  "resources": [
+    "blog_posts:my-first-blog-post",
+    "blog_posts:2",
+    "blog_posts:3"
+  ],
+  "actions": ["delete", "create", "read", "modify"],
+  "effect": "allow"
+}
+```
+
+## Rewriting it to relation tuples
+
+According to the example above we need to create required [namespace](https://www.ory.sh/docs/keto/concepts/namespaces) and [relationship tuple](https://www.ory.sh/docs/keto/concepts/relation-tuples)
+
+General mapping from old to new policies
+
+- Subjects -> Subject/Subject Sets
+- Resources -> Namespaces/Objects
+- Actions -> Relations
+- Effect -> Became obsolete or can be considered as Relations
+
+We need to have `blog_posts` namespace for our example. Let's add the following content to `keto.yml` configuration file. You can find a good template [here](https://github.com/ory/keto/blob/master/config/keto.yml).
```

Comment:
Don't use links to master but always permalinks.
```suggestion
We need to have `blog_posts` namespace for our example. Let's add the following content to `keto.yml` configuration file. You can find a good template [here](https://github.com/ory/keto/blob/3453d47e67cfe45ad1389a2aa25fd65dfdeb114d/config/keto.yml).
```

Maybe rather link to the config reference? https://www.ory.sh/docs/keto/reference/configuration